app/src/main/java/fr/free/nrw/commons/upload/UploadService.java

```diff
@@ -285,7 +286,7 @@ private void uploadContribution(Contribution contribution) {
                         contribution.setFilename(canonicalFilename);
                         contribution.setImageUrl(uploadResult.getImageinfo().getOriginalUrl());
                         contribution.setState(Contribution.STATE_COMPLETED);
-                        contribution.setDateUploaded(CommonsDateUtil.getIso8601DateFormatShort()
+                        contribution.setDateUploaded(CommonsDateUtil.getIso8601DateFormatTimestamp()
```

Comment:
Now that we are saving HH:MM: ss as well, the getter of the dateUploaded in the Contribution class needs to be modified, so that it only returns yyyy-MM-dd for existing components who needed it in this format

Reply:
The return format has been changed